### PR TITLE
Improve download command

### LINKED ISSUE #111: Command "translation:download" should be more verbose
1. I have no translations on the remote end
2. I start the "php bin/console translation:download"
3. Nothing happens (no files created. no output)

A small summary would be enough:

`Files created: 0, translations downloaded: 0`

### PR DESCRIPTION
**Pull request by odolbeau**
Fix #111

### CODE REVIEW

File: Command/DownloadCommand.php [hunk 1]
```diff
@@ -57,31 +56,55 @@ protected function configure(): void
         $this
             ->setName(self::$defaultName)
             ->setDescription('Replace local messages with messages from remote')
+            ->setHelp(<<<EOT
+The <info>%command.name%</info> will erase all your local translations and replace them with translations downloaded from the remote.
+EOT
+            )
             ->addArgument('configuration', InputArgument::OPTIONAL, 'The configuration to use', 'default')
-            ->addOption('cache', null, InputOption::VALUE_NONE, 'Clear the cache if the translations have changed')
+            ->addOption('cache', null, InputOption::VALUE_NONE, '[DEPRECATED] Cache is now automatically cleared when translations have changed.')
```
Comment (odolbeau) [inline Command/DownloadCommand.php]: WDYT about this?

File: Command/DownloadCommand.php [hunk 2]
```diff
@@ -57,31 +56,55 @@ protected function configure(): void
...
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $io = new SymfonyStyle($input, $output);
+
+        if ($input->getOption('cache')) {
+            $message = 'The --cache option is deprecated as it\'s now the default behaviour of this command.';
+
+            $io->error($message);
```
Comment (odolbeau) [inline Command/DownloadCommand.php]: Is it relevant to display an error or is the `trigger_error` call enough?

Comment (rvanlaak) [reply Command/DownloadCommand.php]: Good question, especially as it's on CLI. 

Maybe use `note`? To be sure return code isn't affected?

https://symfony.com/doc/current/console/style.html#admonition-methods

Comment (odolbeau) [reply Command/DownloadCommand.php]: The return code is never affected by what you display in the CLI. :)
With a note:
image
Wit a caution:
image
With an error:
image

My question was: should we display something in the cli regarding the deprecated option or is the `trigger_error` call enough?
I used a note as suggested cause it's less "violent" than a caution or an error but we still have the information directly in the cli which is relevant IMO.

**Pull request merged**